andela/trime
# [Finish #120096931] Add placeholder text

### What this PR does
Adds a placeholder text to show when a project doesn't have
time tracks.

### How can this be manually tested
Add a project to an organisation and then go to that project. You should see a message since you won't have any time logged. If you click on the `start` button, the message should be modified to indicate that you just started tracking time.

### Relevant PivotalTracker stories
[Show a message if the current project has no time tracks logged yet.](https://www.pivotaltracker.com/story/show/120096931)
